instructions/index.js

```diff
@@ -0,0 +1,15 @@
+let today = new Date();
+let thisYear = today.getFullYear();
+let footer = document.querySelector("footer");
+let copyright = document.createElement("p");
+copyright.innerText = "Vanessa Rodriguez " + thisYear;
+footer.appendChild(copyright);
+
+const skills = ["Patient Assessment", "HTML", "CSS", "JavaScript", "Medical Terminology"];
+let skillsSection = document.querySelector("skills");
+let skillsList = document.getElementById("skills")
```

Comment:
Related to the above comment, this selector would actually be returning the `skillsSection`, not the `skillsList`. This function is just a quicker way to find elements by ID (which you can also do using `querySelector` with a `#` in the string. The element with an ID of `skills` in your DOM is `<section id="skills">`, so that's what you'll find here. Since you didn't include the `<ul>` element that you're supposed to be selecting here, there's nothing you can do to find it, but once you include it, that's the one you want to save as `skillsList`